src/Compilers/CSharp/Portable/Symbols/PropertySymbolExtensions.cs

```diff
@@ -70,5 +70,13 @@ public static bool HasRefOrOutParameter(this PropertySymbol property)
             }
             return false;
         }
+
+        public static bool ShouldDisplayReadOnly(this IPropertySymbol property)
+        {
+            // If at least one accessor is present and all present accessors are readonly, the property should be marked readonly.
+            return (property.GetMethod != null || property.SetMethod != null) &&
+                property.GetMethod?.IsReadOnly != false &&
+                property.SetMethod?.IsReadOnly != false;
```

Comment:
Shouldn't this be checking if it's declared readonly vs. `IsReadOnly`? Guessing this is contributing to the diff in the IL suites.

Reply:
Yeah, I thought it would be helpful to display the effective readonly-ness but it seems to be breaking the world too much to do so.

Reply:
All the IL tests that involve declared readonly members will still need to be modified though.